Assets/AddBuildScenes.cs

```diff
@@ -0,0 +1,31 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEditor;
+using UnityEngine;
+
+public class AddBuildScenes : MonoBehaviour
+{
+
+    // Start is called before the first frame update
+    void Start()
+    {
+        List<SceneAsset> m_SceneAssets = new List<SceneAsset>();
+        // Find valid Scene paths and make a list of EditorBuildSettingsScene
+        List<EditorBuildSettingsScene> editorBuildSettingsScenes = new List<EditorBuildSettingsScene>();
+        foreach (var sceneAsset in m_SceneAssets)
```

Comment:
Looks like this doesn't do anything - m_SceneAssets will be an empty list as its only just created.

Reply:
Ah, this one wasn't supposed to be here at all. First attempt at it